src/LivewireManager.php

```diff
@@ -52,6 +52,11 @@ function precompiler($callback)
         app(ExtendBlade::class)->livewireOnlyPrecompiler($callback);
     }
 
+    function isRenderingComponent()
+    {
+        return app(ExtendBlade::class)->isRenderingLivewireComponent();
```

Comment:
What's the performance of this? It looks to me like this is called hundreds / thousands of times when rendering a template, and each time calling `app(ExtendBlade::class)` - is that fast enough to be called that often? Or is there a better way to do this?

Reply:
I should've read @joshhanley's comment above before looking into the code :upside_down_face: That's great, that the performance is only minimally impacted by this!